added instructions for moving baseline, GPS heading & Ublox updating (#1740)
PX4/PX4-user_guide: en/uavcan/ark_rtk_gps.md

### Mounting/Orientation

> The ARK RTK GPS needs to be mounted with the connectors on the board pointing towards the **back of vehicle**, as shown in the following picture.

Reviewer: ```suggestion
The ARK RTK GPS must be mounted with the connectors on the board pointing towards the **back of vehicle**, as shown in the following picture.
```

Setup via CAN:
- Ensure the ARK RTK GPS modules are connected to the Pixhawk via CAN (one can connect to another's secondary CAN port)
- Choose one ARK RTK GPS to be the *Rover* and one to be the *Moving Base*
- Reopen QGroundControl, go to parameters, and select `Standard` to hide that dropdown and select `Component ##` to view each of your ARK RTK GPS's CAN node parameters
:::note
`Component ##` won't be visible unless the ARK RTK GPS is connected to the Pixhawk prior to opening QGroundControl
:::
- On the *Rover*, set the following:
  - [GPS_UBX_MODE](../advanced_config/parameter_reference.md#GPS_UBX_MODE) to `3`
  - [GPS_YAW_OFFSET](../advanced_config/parameter_reference.md#GPS_YAW_OFFSET) to `0` if your *Rover* is in front of your *Moving Base*, `90` if *Rover* is right of *Moving Base*, `180` if *Rover* is behind *Moving Base*, or `270` if *Rover* is left of *Moving Base*
- On the *Moving Base*, set the following:
  - [GPS_UBX_MODE](../advanced_config/parameter_reference.md#GPS_UBX_MODE) to `4`

Setup via UART:

> - Ensure the ARK RTK GPS modules are connected to the Pixhawk via CAN

Reviewer: Do you mean this? i.e. is CAN connection required for the UART setup. I expect so, but just checking

Author: @AlexKlimaj , I believe so right?

Reviewer: Cool. I'll merge anyway and we can update if needed. Thanks for the quick turnaround.
